net/zerotier/src/opnsense/mvc/app/views/OPNsense/Zerotier/index.volt

```diff
@@ -42,37 +49,55 @@ POSSIBILITY OF SUCH DAMAGE.
             }
         );
 
-        ajaxCall(url="/api/zerotier/zerotier/status", sendData={}, callback=function(data,status) {
-            updateServiceStatusUI(data['status']);
+        ajaxCall(url="/api/zerotier/zerotier/status", sendData={}, callback=function(data, status) {
+            updateServiceStatusUI(data['result']);
+            toggleNetworksTab(data['result']);
         });
 
-        $("#reconfigureZerotier").click(function() {
-            $("#reconfigureZerotierProgress").addClass("fa fa-spinner fa-pulse");
-            ajaxCall(url="/api/zerotier/zerotier/reconfigureZerotier", sendData={}, callback=function(data, status) {
-                ajaxCall(url="/api/zerotier/zerotier/status", sendData={}, callback=function(data,status) {
-                    updateServiceStatusUI(data['status']);
+        $("#save").click(function() {
+            $("#saveProgress").addClass("fa fa-spinner fa-pulse");
+            saveFormToEndpoint(url="/api/zerotier/zerotier/set", formid="global", callback_ok=function(data, status) {
+                ajaxCall(url="/api/zerotier/zerotier/status", sendData={}, callback=function(data, status) {
+                    updateServiceStatusUI(data['result']);
+                    toggleNetworksTab(data['result']);
                 });
-                $("#reconfigureZerotierProgress").removeClass("fa fa-spinner fa-pulse");
-                if (status != "success" || data['status'] != 'OK') {
-                    BootstrapDialog.show({
-                        type: BootstrapDialog.TYPE_WARNING,
-                        title: "{{ lang._('Error reconfiguring Zerotier') }}",
-                        message: data['status'],
-                        draggable: true
-                    });
-                }
+                $("#saveProgress").removeClass("fa fa-spinner fa-pulse");
             });
         });
+
+        function toggleNetworksTab(status) {
+            switch(status) {
+                case "disabled":
+                case "service_not_enabled":
```

Comment:
this works perfectly fine without the underscore, just as info

Reply:
Yes, but for readability, I wanted to snakecase the wording. No other reason.